nixos/roles/mysql.nix

```diff
@@ -131,15 +131,26 @@ in
     ];
 
     systemd.tmpfiles.rules = [
-      "d /var/log/mysql 0755 mysql service -"
+      "d /var/log/mysql 0755 mysql service - 90d"
```

Comment:
This doesn't look right. I think the '-' needs to go away as that's the field for the cleanup age?

man tmpfiles.d says:

```
       #Type Path                                     Mode User Group Age         Argument
       d     /directory/to/create-and-cleanup         mode user group cleanup-age -
```